src/test/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSparkIntegrationTest.java

```diff
@@ -0,0 +1,114 @@
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class AlignAssembledContigsSparkIntegrationTest extends CommandLineProgramTest {
+
+    private static final class AlignAssembledContigsSparkIntegrationTestArgs {
+        final String inputAssemblies;
+        final String alignmentOutput;
+        final String alignerRefIndexImgLoc;
+
+        AlignAssembledContigsSparkIntegrationTestArgs(final String inputAssemblies, final String alignmentOutput, final String alignerRefIndexImgLoc){
+            this.inputAssemblies = inputAssemblies;
+            this.alignerRefIndexImgLoc = alignerRefIndexImgLoc;
+            this.alignmentOutput = alignmentOutput;
+        }
+
+        String getCommandLineNoApiKey() {
+            return " --inputAssemblyDir " + inputAssemblies +
+                    " -O " + alignmentOutput +
+                    " --bwamemIndexImage " + alignerRefIndexImgLoc;
+        }
+
+        String getCommandLine() {
+            return  getCommandLineNoApiKey() +
+                    " --apiKey " + getGCPTestApiKey();
+        }
+    }
+
+    @DataProvider(name = "alignAssembledContigsSparkIntegrationTest")
+    public Object[][] createTestData() throws IOException {
+        List<Object[]> tests = new ArrayList<>();
+
+        final File tempWorkingDir = BaseTest.createTempDir("whatever");
+        tempWorkingDir.deleteOnExit();
+
+        final File assemblyWithPackedFastaWithLength = Files.createDirectory(Paths.get(tempWorkingDir.getAbsolutePath()+"/"+"assemblyWithPackedFastaWithLength")).toFile();
+        try(  final PrintWriter out = new PrintWriter( new File(assemblyWithPackedFastaWithLength, "contents.txt") )  ){
+            out.println( ContigCollectionTest.contigsCollectionPackedFastaAndWithAsmId[0][1] );
+        }
+        tests.add(new Object[]{new AlignAssembledContigsSparkIntegrationTest.AlignAssembledContigsSparkIntegrationTestArgs(assemblyWithPackedFastaWithLength.getAbsolutePath(),
+                tempWorkingDir.getAbsolutePath()+"/"+"outputWithLength",
+                SVIntegrationTestDataProvider.ALIGNER_INDEX_IMG)});
+
+        final File assemblyWithPackedFastaWithoutLength = Files.createDirectory(Paths.get(tempWorkingDir.getAbsolutePath()+"/"+"assemblyWithPackedFastaWithoutLength")).toFile();
+        try(  final PrintWriter out = new PrintWriter( new File(assemblyWithPackedFastaWithoutLength, "contents.txt") )  ){
+            out.println( ContigCollectionTest.contigsCollectionPackedFastaAndWithAsmId[0][1] );
+        }
+        tests.add(new Object[]{new AlignAssembledContigsSparkIntegrationTest.AlignAssembledContigsSparkIntegrationTestArgs(assemblyWithPackedFastaWithoutLength.getAbsolutePath(),
+                tempWorkingDir.getAbsolutePath()+"/"+"outputWithoutLength",
+                SVIntegrationTestDataProvider.ALIGNER_INDEX_IMG)});
+
+        return tests.toArray(new Object[][]{});
```

Comment:
Why not add the expected results to this test and compare to make sure the files are equal? Seems like an easy thing to do.